ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml

```diff
@@ -16,6 +16,12 @@ images:
 - dockerfile_path: Dockerfile
   from: base
   to: special-resource-operator
+- dockerfile_literal: |-
```

Comment:
Maybe we should add an image coming from src with patch installed? This one could be used for verify, unit testing and linting.
`golangci-lint` makefile target downloads the tool if needed and we would reduce the possible version drift.

Reply:
`src` already has `patch` installed.